typings/web-inspector.d.ts

```diff
@@ -48,13 +49,16 @@ declare global {
     export interface ParsedURL {
       scheme: string;
       host: string;
+      securityOrigin(): string;
     }
 
     export interface NetworkRequestInitiator {
       type: 'script' | 'parser';
     }
 
     export interface NetworkRequestTiming {
```

Comment:
we should drop `NetworkRequestTiming` altogether since (@paulirish just pointed out) this is actually just `LH.Crdp.Network.ResourceTiming` and is already typed for us :)

You can switch or I can take it later

Reply:
@brendankenny how about you take this later, yah